daemon/rpc_set_defaults.go

```diff
@@ -26,6 +26,16 @@ func (r *RPC) SetDefaults(ctx context.Context, in *pb.Empty) (*pb.Payload, error
 		log.Println(internal.WarningPrefix, err)
 	}
 
+	if err := r.ncClient.Stop(); err != nil {
+		log.Println(internal.WarningPrefix, err)
+	}
+
+	if internal.IsDevEnv(string(r.environment)) {
```

Comment:
same here, isn't needed `!IsProdEnv`?

Reply:
`IsDevEnv` is literaly defined as `!IsProdEnv`:
```go
func IsDevEnv(env string) bool {
	return !IsProdEnv(env)
}
```

Reply:
yes, ignore :smile: